packages/components-js/projects/partials/tests/unit/specs/getMetaTagsAndIconLinks.spec.ts

```diff
@@ -35,16 +35,80 @@ describe('format: jsx', () => {
     );
     expect(renderToString(result)).toMatch(regex);
   });
 
   it('should return meta tags and icon links for china cdn', () => {
     const result = getMetaTagsAndIconLinks({ format: 'jsx', appTitle: 'Porsche UX Platform', cdn: 'cn' });
     const regex = new RegExp(
-      `^<meta name="theme-color" content="#FFF" media="\\(prefers-color-scheme:light\\)"/><meta name="theme-color" content="#0E1418" media="\\(prefers-color-scheme:dark\\)"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="white"/><meta name="apple-mobile-web-app-title" content="Porsche UX Platform"/><meta name="msapplication-TileImage" content="${baseHrefCn}\/mstile-270x270\\.${hash}\\.png"/><meta name="msapplication-TileColor" content="#FFF"/><link rel="icon" sizes="any" href="${baseHrefCn}\/favicon\\.${hash}\\.ico"/><link rel="icon" type="image\/png" sizes="32x32" href="${baseHrefCn}\/favicon-32x32\\.${hash}\\.png"/><link rel="apple-touch-icon" href="${baseHrefCn}\/apple-touch-icon-180x180\\.${hash}\\.png"/><link rel="manifest" href="${baseHrefCn}\/manifest\\.cn\\.${hash}\\.webmanifest"/>$`
+      `^<meta name="theme-color" content="#FFF" media="\\(prefers-color-scheme:light\\)"/><meta name="theme-color" content="#0E1418" media="\\(prefers-color-scheme:dark\\)"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-status-bar-style" content="default"/><meta name="apple-mobile-web-app-title" content="Porsche UX Platform"/><meta name="msapplication-TileImage" content="${baseHrefCn}\/mstile-270x270\\.${hash}\\.png"/><meta name="msapplication-TileColor" content="#FFF"/><link rel="icon" sizes="any" href="${baseHrefCn}\/favicon\\.${hash}\\.ico"/><link rel="icon" type="image\/png" sizes="32x32" href="${baseHrefCn}\/favicon-32x32\\.${hash}\\.png"/><link rel="apple-touch-icon" href="${baseHrefCn}\/apple-touch-icon-180x180\\.${hash}\\.png"/><link rel="manifest" href="${baseHrefCn}\/manifest\\.cn\\.${hash}\\.webmanifest"/>$`
     );
     expect(renderToString(result)).toMatch(regex);
   });
 });
+
+describe('format: js', () => {
+  it('should return meta tags and icon links', () => {
+    const title = 'Porsche UX Platform';
+    const result = getMetaTagsAndIconLinks({ format: 'js', appTitle: title });
+    const expected = {
+      themeColor: [
+        { media: '(prefers-color-scheme: light)', color: '#FFF' },
+        { media: '(prefers-color-scheme: dark)', color: '#0E1418' },
```

Comment:
See above, import, or maybe share the whole config somewhere so it's not tripled

Reply:
For the unit test i would keep it hardcoded so you know when something changes